Dockerfile.stream9

```diff
@@ -18,5 +20,18 @@ RUN dnf install -y \
         gzip \
         make \
         rpm-build \
-        tar
-COPY containerroot /
+        tar \
+
+COPY containerroot /workspace/
+
+# Download source rpms and install it's dependencies
+RUN dnf download --source \
+    ovirt-engine \
+    vdsm \
+    && \
+    dnf builddep -y *.src.rpm
+
+# Do a cleanup of old src rpm and make the container smaller
+RUN rm -f *.src.rpm \
```

Comment:
If you perform this step in a separate `RUN` it is not going to decrease the container size as it will be a separate layer.

Reply:
Hmm you are right, so it needs to be in the same layer I see, thanks

Reply:
Also even if we do `dnf clean all` at the end it does not change much 1.17 vs 1.14 GB which is still nice, but not that important.